go/vt/vttablet/tabletserver/connpool/dbconn_test.go

```diff
@@ -479,18 +448,14 @@ func TestDBNoPoolConnKill(t *testing.T) {
 	if dbConn != nil {
 		defer dbConn.Close()
 	}
-	if err != nil {
-		t.Fatalf("should not get an error, err: %v", err)
-	}
+	require.NoError(t, err)
 	query := fmt.Sprintf("kill %d", dbConn.ID())
 	db.AddQuery(query, &sqltypes.Result{})
 	// Kill failed because we are not able to connect to the database
 	db.EnableConnFail()
 	err = dbConn.Kill("test kill", 0)
-	want := "errno 2013"
-	if err == nil || !strings.Contains(err.Error(), want) {
-		t.Errorf("Exec: %v, want %s", err, want)
-	}
+	require.Error(t, err)
+	require.ErrorContains(t, err, "errno 2013")
```

Comment:
Things like this are probably good places to test our MySQL error handling as this should be `sqlerror.CRServerLost`, no?

Reply:
Fixed